ROADMAP.md

```diff
@@ -8,73 +8,118 @@ not listed now does not mean it will not be considered for Antrea. We definitely
+Kubernetes Network Policies and provides many additional capabilities. One of
+them is the ability to define policy rules using domain names (FQDNs). We think
+there is some room to improve user experience with this feature, and we are
+working on making it more stable.
+
+### Implementation of new upstream NetworkPolicy APIs
+
+[SIG Network](https://github.com/kubernetes/community/tree/master/sig-network)
+is working on [new standard APIs](https://network-policy-api.sigs.k8s.io/) to
+extend the base K8s NetworkPolicy resource. We are closely monitoring the
+upstream work and implementing these APIs as their development matures.
+
+### Better network troubleshooting with packet capture
+
+Antrea comes with many tools for network diagnostics and observability. You may
+already be familiar with Traceflow, which lets you trace a single packet through
+the Antrea network. We plan on also providing users with the ability to capture
+live traffic and export it in PCAP format. Think tcpdump, but for K8s and
+through a dedicated Antrea API!
+
+### Multi-network support for Pods
+
+We recently added the SecondaryNetwork feature, which supports provisioning
+additional networks for Pods, using the same constructs made popular by
+[Multus](https://github.com/k8snetworkplumbingwg/multus-cni). However, at the
+moment, options for network "types" are limited. We plan on supporting new uses
+cases (e.g., secondary network overlays, network acceleration with DPDK), as
+well as on improving user experience for this feature (with some useful
+documentation).
+
+### L7 security policy
+
+Support for L7 NetworkPolicies was added in version 1.10, providing the ability
+to select traffic based on the application-layer context. However, the feature
+currently only supports HTTP traffic, and we plan to extend support to other
+protocols, such as DNS.
+
+### Multi-cluster networking
+
+Antrea can federate multiple K8s clusters, but this feature (introduced in
+version 1.7) is still considered Alpha today. Most of the functionality is
+already there (multi-cluster Services, cross-cluster connectivity,
+and multi-cluster NetworkPolicies), but we think there is some room for
+improvement when it comes to stability and usability.
+
+### NetworkPolicy scale and performance tests
+
+We are working on a framework to empower contributors and users to benchmark the
+performance of Antrea at scale.
+
+### Investigate better integration with service meshes
+
+As service meshes start introducing alternatives to the sidecar approach,
+we believe there is an opportunity to improve the synergy between the K8s
+network plugin and the service mesh provider. In particular, we are looking at
+how Antrea can integrate with the new Istio ambient data plane mode. Take a look
+at [#5682](https://github.com/antrea-io/antrea/issues/5682) for more
+information.
+
+### Investigate multiple replicas for the Controller
+
+While today the Antrea Controller can scale to 1000s of K8s Nodes and 100,000
```

Comment:
```suggestion
While today the Antrea Controller can scale to 1000 of K8s Nodes and 100,000
```

Reply:
That was intentional. I mean _thousands_ of Nodes